pilot/pkg/model/context.go

```diff
@@ -77,9 +77,12 @@ type Proxy struct {
 	// ID is the unique platform-specific sidecar proxy ID
 	ID string
 
-	// Domain defines the DNS domain suffix for short hostnames (e.g.
+	// DNSDomain defines the DNS domain suffix for short hostnames (e.g.
```

Comment:
Not sure this is needed, Domain is consistently used to mean DNS, and TrustDomain is clear. But probably ok.